Multisig.xcodeproj/xcshareddata/xcschemes/Multisig.xcscheme

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1140"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
```

Comment:
I don't understand why this file was added

Reply:
I added some CommandLineArgument to see sql queries in console, so this scheme became shared.